v2/lua/openresty_org/controller.lua

```diff
@@ -207,6 +214,25 @@ function _M.run()
         return
     end
 
+    if tag == "videos" then
+        local main_menu = model.get_main_menu(lang)
+        local timeline = model.get_timeline(lang)
+
+        local videos_html = get_videos_html(lang, i18n);
+
+        local html = view.process("page.tt2",
+                                  { main_menu = main_menu,
```

Comment:
Should we add a line break after left brace?

Reply:
changed.